pulp_container/tests/functional/api/test_rbac_repo_versions.py

```diff
@@ -271,17 +271,18 @@ def setUpClass(cls):
         cls.distribution = cls.distributions_api.list(name="test-1").results[0]
 
         cls.user_pull = gen_user(
-            object_permissions=[
-                ("container.pull_containerdistribution", cls.distribution.pulp_href)
+            object_roles=[
+                ("container.containerdistribution_consumer", cls.distribution.pulp_href),
+                ("container.containernamespace_consumer", cls.distribution.namespace),
             ]
         )
         cls.user_push = gen_user(
-            [
-                "container.push_containerdistribution",
-                "container.namespace_push_containerdistribution",
+            object_roles=[
+                ("container.containerdistribution_collaborator", cls.distribution.pulp_href),
+                ("container.containernamespace_collaborator", cls.distribution.namespace),
```

Comment:
Again, i think one of them should be sufficient.

Reply:
Yes, you are right. I tested the push permission with the namespace permission and it was working.

Reply:
Now i'm asking myself whether we should have a test for both individually.

Reply:
The test case tries to verify whether a user has all the required permissions to perform blob mounting (`pull` for one repository & `push` for another repository). We are not trying to test the whole access_policy workflow for pushing content. I am fine with namespace permissions only.

Reply:
Maybe, I should move the test case to another file. The name evokes that we are trying to test the whole RBAC suite, which is wrong.

Reply:
Well, we are creating users and so we testing for rbac features. I think, the name fits.